adapters/adtelligent/adtelligent.go

```diff
@@ -171,7 +171,8 @@ func validateImpression(imp *openrtb2.Imp) (int, error) {
 	// common extension for all impressions
 	var impExtBuffer []byte
 
-	impExtBuffer, err = json.Marshal(&adtelligentImpExt{
+	// TODO: confirm the missing error handling is intended
+	impExtBuffer, _ /* err */ = json.Marshal(&adtelligentImpExt{
```

Comment:
Just want to circle back to this to make sure the error is being properly handled and bubbled up, and that we can delete the TODO on line 174

Reply:
ack - changed to:
```go
	impExtBuffer, err = json.Marshal(&adtelligentImpExt{
		Adtelligent: impExt,
	})
	if err != nil {
		return 0, &errortypes.BadInput{
			Message: fmt.Sprintf("ignoring imp id=%s, error while marshaling impExt, err: %s", imp.ID, err),
		}
	}
```